beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/BlockFactoryDenebTest.java

```diff
@@ -89,6 +93,72 @@ void unblindSignedBlock_shouldUnblindBeaconBlock() {
     assertThat(unblindedBlock).isEqualTo(expectedUnblindedBlock);
   }
 
+  @Test
+  void shouldCreateValidBlobSidecarsForBlockContents() {
+    final Spec spec = TestSpecFactory.createMinimalDeneb();
+    final BlobsBundle blobsBundle = prepareBlobsBundle(spec, 3);
+
+    final BlockAndBlobs blockAndBlobs = assertBlobSidecarsCreated(false, spec);
+
+    final List<BlobSidecar> blobSidecars = blockAndBlobs.blobSidecars();
```

Comment:
could you please add assert not empty here, I'm a bit paranoid

Reply:
This is done in the `assertBlobSidecarsCreated` method following the pattern from other unit tests in this class.